gulpfile.babel.js

```diff
@@ -1,16 +1,27 @@
 const Bundle  = require('./bundle')
+const cfgstr  = require('configstore')
 const babel   = require('gulp-babel')
 const concat  = require('gulp-concat')
 const del     = require('del')
 const gulp    = require('gulp')
+const inq     = require('inquirer')
 const jade    = require('gulp-jade')
 const mocha   = require('gulp-mocha')
 const nib     = require('nib')
 const path    = require('path')
+const pkg     = require('./package.json')
 const plumber = require('gulp-plumber')
+const replace = require('gulp-replace-task')
+const prompts = require('./setup-prompts')
 const stylus  = require('gulp-stylus')
 
 const config = {
+  db: new cfgstr(
+    pkg.name,
+    {
+      "isFirstRun": true
```

Comment:
JavaScript styleguides ([Google](http://google.github.io/styleguide/javascriptguide.xml#Strings), [Node.js](https://github.com/felixge/node-style-guide#use-single-quotes) and [Airbnb](https://github.com/airbnb/javascript#6.1)) prefer single-quotes over doubles, could you change them?